api/python/notebooks/api_demo/census_citation_generation.ipynb

```diff
@@ -0,0 +1,331 @@
+   "id": "941c6f2d-6bc2-46c3-963e-e74335fe93f6",
+   "metadata": {},
+   "source": [
+    "### Via AnnData query"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "2d9b2a11-2f48-43a5-8955-759019ce6bed",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Publication: https://doi.org/10.1126/science.abl4896 Dataset Version: https://datasets.cellxgene.cziscience.com/4866a804-37eb-436f-8c87-9cd585260061.h5ad curated and distributed by CZ CELLxGENE Discover in Collection: https://cellxgene.cziscience.com/collections/e5f58829-1a66-40b5-a624-9046778e74f5\n",
+      "\n",
+      "Publication: https://doi.org/10.1126/science.abl4896 Dataset Version: https://datasets.cellxgene.cziscience.com/bfd80f12-725c-4482-ad7f-1ed2b4909b0d.h5ad curated and distributed by CZ CELLxGENE Discover in Collection: https://cellxgene.cziscience.com/collections/e5f58829-1a66-40b5-a624-9046778e74f5\n",
+      "\n",
+      "Publication: https://doi.org/10.1126/science.abl4896 Dataset Version: https://datasets.cellxgene.cziscience.com/e6df8a57-f54f-413a-9d4d-dee03294d778.h5ad curated and distributed by CZ CELLxGENE Discover in Collection: https://cellxgene.cziscience.com/collections/e5f58829-1a66-40b5-a624-9046778e74f5\n",
+      "\n",
+      "Publication: https://doi.org/10.1126/science.abl4896 Dataset Version: https://datasets.cellxgene.cziscience.com/8d599205-5c51-4b50-9d48-3dec31238587.h5ad curated and distributed by CZ CELLxGENE Discover in Collection: https://cellxgene.cziscience.com/collections/e5f58829-1a66-40b5-a624-9046778e74f5\n",
+      "\n",
+      "Publication: https://doi.org/10.1126/science.abl4896 Dataset Version: https://datasets.cellxgene.cziscience.com/f6065c51-bd26-4aa5-a05d-2805aeea48d9.h5ad curated and distributed by CZ CELLxGENE Discover in Collection: https://cellxgene.cziscience.com/collections/e5f58829-1a66-40b5-a624-9046778e74f5\n",
+      "\n",
+      "Publication: https://doi.org/10.1126/science.abl4896 Dataset Version: https://datasets.cellxgene.cziscience.com/8cdbf790-4d29-4f46-9aef-21adfb2e21da.h5ad curated and distributed by CZ CELLxGENE Discover in Collection: https://cellxgene.cziscience.com/collections/e5f58829-1a66-40b5-a624-9046778e74f5\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Fetch an AnnData object\n",
+    "adata = cellxgene_census.get_anndata(\n",
+    "    census=census,\n",
+    "    organism=\"homo_sapiens\",\n",
+    "    measurement_name=\"RNA\",\n",
+    "    obs_value_filter=\"tissue == 'cardiac atrium'\",\n",
+    "    var_value_filter=\"feature_name == 'MYBPC3'\",\n",
+    "    column_names={\"obs\": [\"dataset_id\", \"cell_type\"]},\n",
+    ")\n",
+    "\n",
+    "# Get a citation string for the slice\n",
+    "slice_datasets = datasets[datasets[\"dataset_id\"].isin(adata.obs[\"dataset_id\"])]\n",
+    "print(*slice_datasets[\"citation\"], sep=\"\\n\\n\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
```

Comment:
This cell should probably move after the `.shape` one.

Reply:
This one was for testing, I removed it.